pipelines/legionPipeline.groovy

```diff
@@ -210,9 +104,9 @@ def runRobotTestsAtGcp(tags="") {
                     wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
                         docker.image("${env.param_docker_repo}/k8s-terraform:${env.param_legion_infra_version}").inside("-e GOOGLE_CREDENTIALS=${gcpCredential} -e CLUSTER_NAME=${env.param_cluster_name} -u root") {
                             stage('Extract Hiera data') {
-                                extractHiera("json")
-                                gcp_zone = sh(script: "jq '.location' ${WORKSPACE}/cluster_profile.json", returnStdout: true)
-                                gcp_project_id = sh(script: "jq '.project_id' ${WORKSPACE}/cluster_profile.json", returnStdout: true)
+                                extractHiera()
+                                gcp_zone = sh(script: "jq '.location' ${env.clusterProfile}", returnStdout: true)
```

Comment:
I would like to move the extraction of these variables(gcp_zone, gcp_project_id) to the setupGcpAccess function. Otherwise, it is tough to understand.